api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -368,7 +368,8 @@ private void addFieldValuesFromBigQueryToPreviewList(
               valuePreviewList
                   .get(columnNumber)
                   .addQueryValueItem(
-                      Optional.ofNullable(fieldValueList.get(columnNumber).getValue().toString())
+                      Optional.ofNullable(fieldValueList.get(columnNumber).getValue())
+                          .map(Object::toString)
```

Comment:
is this something we can easily add a test for? If its low effort because of existing testing infrastructure, I think it's worth adding.

Reply:
+1 I think . `FieldValueUtils` class to do some of these mappings would be more readable and testable than doing it inline in this controller.

Reply:
It isn't particularly low effort, unfortunately. :( We have some basic bigquery stubbing, but not a ton, and not with much detail. I can poke around at improving that, but want to get this fix in to un-break things in the time being.

Reply:
I may be able to test just this function by constructing a fieldvaluelist? Which seems less valuable but faster to implement. Any thoughts on that?